openmls/src/group/mls_group/proposal.rs

```diff
@@ -384,4 +386,51 @@ impl MlsGroup {
 
         Ok((mls_message, proposal_ref))
     }
+
+    /// Updates group context extensions
+    ///
+    /// Returns an error when the group does not support all the required capabilities
+    /// in the new `extensions`.
+    #[allow(clippy::type_complexity)]
+    pub fn update_group_context_extensions<Provider: OpenMlsProvider>(
+        &mut self,
+        provider: &Provider,
+        extensions: Extensions,
+        signer: &impl Signer,
+    ) -> Result<
+        (MlsMessageOut, Option<MlsMessageOut>, Option<GroupInfo>),
+        CreateGroupContextExtProposalError<Provider::StorageError>,
+    > {
+        self.is_operational()?;
+
+        // Create group context extension proposals
+        let inline_proposals = vec![Proposal::GroupContextExtensions(
+            GroupContextExtensionProposal { extensions },
+        )];
+
+        let params = CreateCommitParams::builder()
+            .framing_parameters(self.framing_parameters())
+            .proposal_store(&self.proposal_store)
+            .inline_proposals(inline_proposals)
+            .build();
+        let create_commit_result = self.group.create_commit(params, provider, signer)?;
+
+        let mls_messages = self.content_to_mls_message(create_commit_result.commit, provider)?;
+        self.group_state = MlsGroupState::PendingCommit(Box::new(PendingCommitState::Member(
+            create_commit_result.staged_commit,
+        )));
+
+        provider
+            .storage()
+            .write_group_state(self.group_id(), &self.group_state)
+            .map_err(CreateGroupContextExtProposalError::StorageError)?;
```

Comment:
This was the fix that fixed our tests. I had forgotten to add this to this method.